[DOC] Edits to connector/task restart operations doc (#4427)
strimzi/strimzi-kafka-operator: documentation/modules/proc-manual-restart-mirrormaker2-connector.adoc

.Procedure

. Find the name of the `KafkaMirrorMaker2` custom resource that controls the Kafka MirrorMaker 2.0 connector you want to restart:
+
[source,shell,subs="+quotes"]
----
kubectl get KafkaMirrorMaker2
----

. Find the name of the Kafka MirrorMaker 2.0 connector to be restarted from the `KafkaMirrorMaker2` custom resource.
+
[source,shell,subs="+quotes"]
----

> kubectl describe KafkaMirrorMaker2 _KAFKAMIRRORMAKER-2-NAME_

Author: Would it be helpful to show the output of this command?

Reviewer: I don't think that is necessary.

----

. To restart the connector, annotate the `KafkaMirrorMaker2` resource in Kubernetes. 

> For example, using `kubectl annotate` to restart the connector named `+my-source->my-target.MirrorSourceConnector+`:

Reviewer 2: ```suggestion
In this example, `kubectl annotate` restarts a connector named `+my-source->my-target.MirrorSourceConnector+`:
```
